server/mongodb/models/embedded/Address.ts

```diff
@@ -15,7 +15,7 @@ const AddressSchema = new Schema<Address>({
     required: true,
   },
   zipCode: {
```

Comment:
I understand the thought process behind making this a number, but we need to leave it as a string to avoid some edge cases.